package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-kubeless",
-  "version": "0.1.11",
```

Comment:
There might be a way to update version automatically with ${TRAVIS_TAG} env

Reply:
Note that `TRAVIS_TAG` is available in the builds triggered by a new tag. These scripts are meant to generate that tag so the environment variable is not available yet.